Source/WebCore/platform/graphics/gstreamer/GStreamerRegistryScanner.cpp

```diff
@@ -1166,20 +1166,20 @@ void GStreamerRegistryScanner::fillVideoRtpCapabilities(Configuration configurat
             element = gst_element_factory_make("webkitvideoencoder", nullptr);
 
         if (element) {
-            Vector<ASCIILiteral> profiles = {
-                "42e01f"_s,
-                "640c1f"_s,
-                "42001f"_s,
-                "4d001f"_s,
+            static Vector<std::pair<ASCIILiteral, unsigned>> profiles = {
```

Comment:
This should not be a vector.
```suggestion
            static constexpr std::array<std::pair<ASCIILiteral, unsigned>> profiles = {
```